lib/modules/apostrophe-schemas/public/js/user.js

```diff
@@ -1176,7 +1176,7 @@ apos.define('apostrophe-schemas', {
       convert: function(data, name, $field, $el, field, callback) {
         data[name] = $field.val();
         // Seems odd but sometimes used to mandate an "I agree" box
-        if (field.required && !data[name]) {
+        if (field.required && (!data[name]) || data[name] === '0') {
```

Comment:
I think you need more parenthesis grouping here, it's ambiguous what's going on between the `&&` and `||`.